lnrpc/walletrpc/walletkit_server.go

```diff
@@ -473,6 +473,13 @@ func (w *WalletKit) SendOutputs(ctx context.Context,
 		})
 	}
 
+	// Then, we'll extract the minimum number of confirmations that each
+	// output we use to fund the transaction should satisfy.
+	minConfs, err := extractSendOutputsMinConfs(req)
```

Comment:
We should share this method between all RPCs having the same `spend_unconfirmed/min_confs` params (including `OpenChannel`).

Reply:
update